src/main/java/org/testng/IAnnotationTransformer3.java

```diff
@@ -4,6 +4,7 @@
 
 public interface IAnnotationTransformer3 extends IAnnotationTransformer2 {
```

Comment:
Same case here as well. Please move the methods in this interface to the parent interface and have this deprecated.